tests/core/editable/removemultiplelist.js

```diff
@@ -0,0 +1,116 @@
+/* bender-ckeditor-plugins: list,toolbar,table,format */
+
+( function() {
+	'use strict';
+
+	bender.editors = {
+		classic: {},
+		inline: {
+			creator: 'inline'
+		}
+	};
+
+	var tests = {
+		'test remove full multiple list': function( editor, bot ) {
+			bender.tools.testInputOut( 'multiple_full_list', function( source, expected ) {
+				bender.tools.setHtmlWithSelection( editor, source );
+
+				editor.editable().fire( 'keydown', new CKEDITOR.dom.event( { keyCode: 46 } ) );
```

Comment:
It would be easier to understand what is happening, if we extract hardcoded `46` into a separate variable, like `DELETE_KEY_CODE` or similar.

Also, is there any difference between using the delete key and backspace? Maybe we should verify the behavior for both of them?

Reply:
I didn't notice any difference when deleting lists using `Delete` or `Backspace` key, but to be sure I added tests for both cases.